package.json

```diff
@@ -15,7 +16,7 @@
     "node": ">=12.13.0"
   },
   "dependencies": {
-    "@elastic.io/object-storage-client": "1.0.1",
+    "@elastic.io/object-storage-client": "2.0.0",
```

Comment:
version 2.0.0 has breaking change and test should fail in addAsStream. But 2.1.0 has reverted it. Use 2.1.0.